### PR TITLE
LPS-181595 Uniform Automated Test | Functional Automation for LPS-181594 - New Property - DL size timeout

### PR DESCRIPTION
**Pull request by lucasperj**

### CODE REVIEW

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:upgrade

**Comment by vicnate5** [comment]
@lucasperj reopen or resend when this is ready.

**Comment by lucasperj** [comment]
ci:reopen

**Comment by lucasperj** [comment]
ci:test:upgrade

File: modules/apps/portal/portal-upgrade-test/src/testIntegration/java/com/liferay/portal/upgrade/test/BaseUpgradeReportLogAppenderTestCase.java [hunk 1]
```diff
@@ -228,10 +228,14 @@ public void run() {
...
 
+			Assert.assertTrue(
+				_getLogContent().contains(
+					"INFO - Unable to determine the document library size. " +
+						"Increase the timeout or check it manually."));
 			_assertLogContextContains(
 				"upgrade.report.document.library.storage.size",
 				"Unable to determine");
-			_assertReport("Unable to determine");
+			_assertReport("Document library storage size: Unable to determine");
```
Comment (lucasperj) [inline modules/apps/portal/portal-upgrade-test/src/testIntegration/java/com/liferay/portal/upgrade/test/BaseUpgradeReportLogAppenderTestCase.java:238]: @vicnate5 , Sofia added the full string assert for case 3

**Comment by lucasperj** [comment]
ci:stop

**Pull request closed without merging**